core/src/main/scala/com/yahoo/maha/core/query/BigqueryHivePrestoQueryCommon.scala

```diff
@@ -132,7 +132,7 @@ method to crawl the NoopRollup fact cols recursively and fill up the parent colu
  whose dependent source columns is/are NoopRollup column.
  All such parent noop rollup columns has to be rendered at OuterGroupBy layer
  */
-  def dfsNoopRollupCols(fact:Fact, cols: Set[(Column, String)], parentList: List[(Column, String)], noopRollupColSet: mutable.LinkedHashSet[(String, Column)]): Unit = {
```

Comment:
Why are we changing from Set to List?  I don't see anywhere the List change is benefiting us.

Reply:
I would expect we only want to use distinct for order preservation, but I don't see that functionality in use.

Reply:
while using Set the generated query will have random column rendering sequence, since the input in each query generator is using toSet() to convert from list to set